docs/tools/opensbi-kernel-for-xs.md

```diff
@@ -0,0 +1,87 @@
+# Linux Kernel for XiangShan in EMU
+
+!!! note
+    注意，下文所有使用的工具链是通过 [riscv-toolchains](https://github.com/riscv-collab/riscv-gnu-toolchain) 构建得到的，要构建在 Linux 上运行的程序使用的前缀是 `riscv64-unknown-linux-gnu-` ，某些 Linux 发行版使用包管理工具安装的工具链仅有 `riscv64-linux-gnu-` 前缀，此时仅需要将下文所有的 `riscv64-unknown-linux-gnu-` 替换为 `riscv64-linux-gnu-` 即可，它和 `riscv64-unknown-linux-gnu-` 是等价的，详细请看[Target Triplet](https://wiki.osdev.org/Target_Triplet)
+
+#### 1. 如何构建一个可在香山或者 NEMU 上运行的使用 OpenSBI 的最简 Linux Kernel
+
+!!! note
+    注意，过程中出现任何问题，请首先观察出错信息并结合 Makefile 文件，自行网上搜索可以解决一些简单问题，比如工具链和依赖包相关的问题。
+
+- 分别克隆或下载启动加载程序 OpenSBI、Linux kernel、根文件系统 rootfs、NEMU以及构建时所需的设备树和内核配置文件。并将四个仓库和一个压缩文件放到**同一目录下**
+```bash
+git clone https://github.com/riscv-software-src/opensbi.git
+wget https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.6.30.tar.xz
+git clone https://github.com/OpenXiangShan/riscv-rootfs -b checkpoint
```

Comment:
I check the `checkpoint` branch and find it just a few commits ahead of the master branch. It would be better to upstream these changes to master.

We can remove unused apps in Makefile. Most users are using rootfs to build SPECCPU2006. We can remove the unused dependencies in Makefile as the `checkpoint` branch does.

Reply:
https://github.com/OpenXiangShan/riscv-rootfs/pull/4

Reply:
Approved and merged